index.js

```diff
@@ -86,8 +67,32 @@ function sss(val) {
 		case "undefined":
 			return null;
 		case "string":
-			return '"' + val.replace(strReg, strReplace) + '"';
+			return '"' + strEscape(val) + '"';
 		default:
 			return val;
 	}
 }
+
+function strEscape(str){
+	var length = str.length,
+		buffer = '',
+		code = 0,
+		i = 0;
+
+	for (; i < length; i++) {
+		code = str.charCodeAt(i);
+
+		if (code === 34) buffer += '\\"';
+		else if (code === 92) buffer += '\\\\';
+		else if (code > 31) buffer += String.fromCharCode(code);
```

Comment:
This is intentionally moved here to optimise the common path, which I suspect is non-whitespace characters.